test/terraform_aws_lambda_test.go

```diff
@@ -0,0 +1,49 @@
+package test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	// "github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAWSLambda(t *testing.T) {
+	// Construct the terraform options with default retryable errors to handle the most common
+	// retryable errors in terraform testing.
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// Set the path to the Terraform code that will be tested.
+		TerraformDir: "../",
+	})
+
+	// Clean up resources with "terraform destroy" at the end of the test.
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Run "terraform init" and "terraform apply". Fail the test if there are any errors.
+	terraform.InitAndApply(t, terraformOptions)
+
+	// Run `terraform output` to get the values of output variables and check they have the expected values.
+	lambdaArn := terraform.Output(t, terraformOptions, "arn")
+	assert.Equal(t, "arn:aws:lambda:us-east-1:000000000000:function:use1-development-testLambda", lambdaArn)
+	// Obtain working directory
+	dirPath, err := os.Getwd()
+	if err != nil {
+	}
```

Comment:
🔧 It's more idiomatic go to just ignore the err return if it's not going to be used. Are we sure that we can ignore the error here if we're expecting it to be valid in the `FileExists` call below?
```suggestion
	dirPath, _ := os.Getwd()
```